src/AppBundle/Form/PostType.php

```diff
@@ -32,14 +32,16 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         // for the full reference of options defined by each form field type
         // see http://symfony.com/doc/current/reference/forms/types.html
         $builder
-            ->add('title')
-            ->add('summary', 'textarea')
+            ->add('title', null, ['label' => 'label.title'])
```

Comment:
Good catch! I'm going to check of non-PHP 5.3 features and I'm going to setup a `.travis.yml` file so we can check on any PHP version. Thanks!